### PR TITLE
Fix state change problems.  Fix typo in logout code.

### PR DESCRIPTION
**Pull request by bramski**
**Overall**
This fixes the state change success problems by issuing a $timeout with $apply.
This also fixes the typo in logout.

### CODE REVIEW

File: app/scripts/services/endpoint.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ endpointClient.factory('Endpoint', function($rootScope, AccessToken, $q, $http)
     AccessToken.destroy();
     $rootScope.$broadcast('oauth:logging-out');
     if( params.logoutPath ) {
-      window.location.replace(buildOauthUrl(params.logOutPath, params));
+      window.location.replace(buildOauthUrl(params.logoutPath, params));
```
Comment (bramski) [inline app/scripts/services/endpoint.js]: Typo.

**Comment by bramski** [comment]
Fixes: https://github.com/angularjs-oauth/oauth-ng/issues/118

**Comment by bramski** [comment]
Fixes: https://github.com/angularjs-oauth/oauth-ng/issues/117

**Comment by m00s** [comment]
@bramski Good catches. Thanks!

**Pull request merged**